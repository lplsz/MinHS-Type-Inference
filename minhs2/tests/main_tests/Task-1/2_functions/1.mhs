main = 
      let f = recfun f x y = x + y ;
    in f 1 2;
