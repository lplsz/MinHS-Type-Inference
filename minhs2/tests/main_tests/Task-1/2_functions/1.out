main :: Int = let f :: (Int -> Int -> Int) = (recfun f :: (Int -> Int -> Int) x y = (+) x y);
              in f 1 2;
